AntWars/AI/AIAntBase.cs

```diff
@@ -20,6 +20,8 @@ public abstract class AIAntBase : IAIAnt {
 
         internal Config Conf { get; set; }
 
+        private static readonly Random randomInstance = new Random();
```

Comment:
Hatte es wegen `private` unter internal eingeordnet.

Hatten wir nicht in irgend einem Issue ne Konvention zu Anordnung?